### PR TITLE
add external module healthy status page

### PR DESCRIPTION
**Pull request by gaertnermarkus**
Display checks for all modules on one page.

### CODE REVIEW

File: views/admin/lt/ac_module_internalls_lang.php [hunk 1]
```diff
@@ -18,6 +18,8 @@
     'tbclmodule_internals_state'    => 'Būklė',
     'tbclmodule_internals_utils'    => 'Įrankiai',
 
+    'SHOP_MODULE_GROUP_AC_CONFIG'    => 'конфигурация',
```
Comment (ddpkts) [inline views/admin/lt/ac_module_internalls_lang.php]: should be `Konfigūracija` ;)

File: views/admin/ru/ac_module_internalls_lang.php [hunk 2]
```diff
@@ -18,6 +18,8 @@
     'tbclmodule_internals_state'    => 'Статус',
     'tbclmodule_internals_utils'    => 'Утилиты',
 
+    'SHOP_MODULE_GROUP_AC_CONFIG'    => 'Config',
```
Comment (ddpkts) [inline views/admin/ru/ac_module_internalls_lang.php]: and `Конфигурация` here

File: views/admin/lt/ac_module_internalls_lang.php (hunk 1 above)
Comment (gaertnermarkus) [reply views/admin/lt/ac_module_internalls_lang.php]: fixed in 52370a2

File: views/admin/ru/ac_module_internalls_lang.php (hunk 2 above)
Comment (gaertnermarkus) [reply views/admin/ru/ac_module_internalls_lang.php]: fixed in 52370a2

**Pull request merged**